skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevvelger/route.tsx

```diff
@@ -62,12 +63,15 @@ export function BrevvelgerPage() {
         background-color: white;
       `}
     >
-      <BrevvelgerMainContent
-        alleSaksbrevQuery={alleSaksbrevQuery}
-        letterTemplates={letterTemplates}
-        saksId={saksId}
-        setOnSubmitClick={setOnSubmitClick}
-      />
+      <LandDataProvider>
```

Comment:
Hva er grunnen til å ha en context-provider for landData og ikke bare useLandData (useQuery) i komponenten?